autoscheduler/frontend/src/components/CourseSelectColumn/CourseSelectColumn.css

```diff
@@ -0,0 +1,36 @@
+.container {
+    width: 20%;

```

Comment:
Before we merge this with master, let's make this width 100%. That way, I can put it inside of a container when I'm composing the master layout and let that dictate the width of the column

Reply:
Yeah that makes sense. I'll leave it how it is rn and will remove it at the end (along with the `/course` route definition in `App.tsx`)

Reply:
Also in that same vein I also need to change the `CourseSelectColumn` to not use the `RouteComponentProps` since it's only needed to be a route for this PR.